src/frameworks/react-i18next.ts

```diff
@@ -33,6 +33,14 @@ class ReactI18nextFramework extends Framework {
     '\\Wi18nKey=[\'"`]({key})[\'"`]',
   ]
 
+  supportAutoExtraction = [
+    'javascript',
+    'typescript',
+    'javascriptreact',
+    'typescriptreact',
+    'html',
+  ]
```

Comment:
Should we move this into the shopify-i18next framework? Otherwise, if it is meant to be in the react-i18next framework, should it be in a separate PR?